### ISSUE TITLE
550 Onion address collision in v4.9.9 when creating new tenants

### ISSUE THREAD

**Issue by aetdr** [globaleaks/GlobaLeaks#3241]
Hi @evilaliv3 

We get this error when creating new tenants with version 4.9.9

```

Version: 4.9.9

txtorcon.torcontrolprotocol.TorProtocolError 
    Happens on 500-level responses in the protocol, almost certainly
    in an errback chain.

    :ivar code: the actual error code
    :ivar text: other text from the protocol
    

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/txtorcon/torconfig.py", line 474, in add_to_tor
    ans = yield protocol.queue_command(cmd)

txtorcon.torcontrolprotocol.TorProtocolError: 550 Onion address collision

```

**Comment by evilaliv3** [comment]
Thank you @aetdr 

We are aware of this exception that apparently is not causing any particular issue except noise.

We are working on fixing and we will issue a release as soon that a fix will be available.

**Comment by evilaliv3** [comment]
This should be fixed in release 4.10.0
